docs/reference-guides/rancher-security/rancher-security-best-practices.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Rancher Security Best Practices
+---
+
+<head>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/reference-guides/rancher-security/rancher-security-best-practices"/>
+</head>
+
+### Restricting public access to /version path
+
+It is well-known that attackers might want to gather any information related to services running online. When running a Rancher server, the `/version` path can disclose some information about the server. We know that this path can be commonly used to monitor uptime from the service or any other monitoring metrics. In this case, we recommend customers block the `/version` path at the infrastructure level, through Firewall rules, keeping it accessible only for monitoring tools or intranet.
```

Comment:
Here's some suggestions on further detailing the issue and the mitigation:

```suggestion
The Rancher Manager server provides information about the version it is running and the Go version that was used to built it. That information is accessible via the `/version` path, and is generally used to automate version bumps, confirm a deployment was successful, amongst other things.

Such information can be used by adversaries to identify the running version and cross relate it with potential bugs that it may have. In cases where the Rancher server is publicly available through the internet, it is recommended that the path `/version` be blocked by using a Layer 7 firewall.
```

Feel free to amend it or drop parts of it.